app/models/quota.rb

```diff
@@ -18,8 +20,39 @@ def find(quota_path, user)
       user  = user && user.to_s
 
       quotas = []
+
+      # Attempt to convert path into a URI
+      if quota_path.instance_of?(String) and quota_path.match(/^https?:/)
+        uri = URI.parse(quota_path)
+        # If it is a URI, and it is http:// or https://
+        begin
+          raw = Net::HTTP.get(uri)
+        rescue Exception => e
```

Comment:
Its best to rescue `StandardError` instead of `Exception`. Rescuing `Exception` will rescue exceptions we don't want to rescue like `NoMemoryError`, `SyntaxError` `SignalException` etc.

Reply:
Thanks, this is helpful. I'm new to Ruby so still learning.